### PR TITLE
Recommend empty project over win32

### PR DESCRIPTION
**Pull request by fahlmant**
To resolve issue #14

### CODE REVIEW

**Comment by LaurentGomila** [comment]
> To resolve issue #14

If the user ticks the "empty project" option of the Win32 project wizard, as clearly mentioned in the tutorial, then the project is indeed empty (no stdafx). And unlike what was said in #14, the entry point remains `main()`. The only difference is that the linker is properly configured for a graphical application, and users won't come to the forum asking how to hide the console.

**Comment by fahlmant** [comment]
If the current documentation is viewed as sufficient, then #14 should be closed

**Comment by eXpl0it3r** [comment]
The problem is that "Win32 project wizard" doesn't exist anymore. When I say "create new project" in VS 2017, I'm greeted with this dialog.

image

I'd suggest to just always recommend an empty project and if they want to "remove the console window", teach them how to set the system variable thingy to 'windows' and add sfml-main to the linker.

**Comment by LaurentGomila** [comment]
Wasn't it just renamed to "Windows Desktop Application", or is it a different thing?

**Comment by eXpl0it3r** [comment]
The phrasing of the tutorial right now is:

> The first thing to do is choose what kind of project to create: you must select **"Win32 application"**. The wizard offers a few options to customize the project: select **"Console application"** if you need the console, or **"Windows application"** if you don't want it. Check the **"Empty project"** box if you don't want to be annoyed with auto-generated code.

In the current wizard there's no category called "Win32 application", as such the phrasing is confusing (there's no "Win32 application, what should I pick now?). The Console application" or "Windows application" can more easily be matched to "Windows Console Application" and "Windows Desktop Application", however I'd argue, that you pretty much never want to recommend (beginners) to pick either of these options, as both will add the pre-compiled header `pch.h` which will cause more confusion than help.

In short: It's easier to teach people how to switch the subsytem than to explain to them how they have to used a pre-compiled header or how to get rid of it. As such I would always recommend "Empty Project".

**Comment by LaurentGomila** [comment]
I don't know VS 2017. What I asked was if "Windows Desktop Application" was just a rename of "Win32 Application", or a totally different thing. In the former case, we just need to reword the tutorial.

> both will add the pre-compiled header pch.h

With VS 2015 and older, checking the "empry project" box (not to be confused with the "Empty Project" project choice) removed the precompiled header. Again, I don't know if still applies to VS 2017.

I'm arguing but in the end I don't care, maybe it's indeed better to change to "Empty project" ; it will also make maintaining the tutorial easier as things change in newer versions of VS.

**Comment by eXpl0it3r** [comment]
Well again, there's no "Win32 Application" and no "Empty Project" checkbox. There's no old style setup wizard. The picture I posted is all you get. The tutorial is based on like VS2005 or VS2008? where stuff like that existed.

**Comment by LaurentGomila** [comment]
The exact same stuff still exists in VS 2015 😉 

Ok, so let's just go with empty project.

File: tutorials/2.5/start-vc.php [hunk 1]
```diff
@@ -31,9 +31,8 @@
 
 <?php h2('Creating and configuring a SFML project') ?>
 <p>
-    The first thing to do is choose what kind of project to create: you must select "Win32 application". The wizard offers a few options to customize
-    the project: select "Console application" if you need the console, or "Windows application" if you don't want it. Check the "Empty project" box if
-    you don't want to be annoyed with auto-generated code.<br />
+    The first thing to do is choose what kind of project to create. It is recommended to select "Empty Project". The wizard offers a few options to customize
+    the project: select "Console application" if you need the console, or "Windows application" if you don't want it.<br />
```
Comment (eXpl0it3r) [inline tutorials/2.5/start-vc.php]: I would use:

> The first thing to do is choose what kind of project to create. It is recommended to select "**Empty Project**". The dialog offers a few other options to customize the project: select "Console application" or "Windows application" only if you know how to use pre-compiled headers.

**Comment by LaurentGomila** [comment]
In french:

> La première chose à faire est de choisir quel type de projet créer. Il est recommandé de sélectionner "Empty project". L'assistant offre quelques options pour personnaliser le projet : choisissez "Console application" ou "Windows application" uniquement si vous savez vous servir des en-têtes précompilés.

I kept english names for the mentioned options, like in the current tutorial. I can't check their french translation, anyway.
